src/libs/auth.js

```diff
@@ -110,24 +131,26 @@ export const initializeAuth = _.memoize(async () => {
         fenceStatus: isSignedIn ? state.fenceStatus : {},
         // Load whether a user has input a cookie acceptance in a previous session on this system,
         // or whether they input cookie acceptance previously in this session
-        cookiesAccepted: isSignedIn ? state.cookiesAccepted || getLocalPrefForUserId(user.getId(), cookiesAcceptedKey) : undefined,
+        cookiesAccepted: isSignedIn ? state.cookiesAccepted || getLocalPrefForUserId(userId, cookiesAcceptedKey) : undefined,
         isTimeoutEnabled: isSignedIn ? state.isTimeoutEnabled : undefined,
         user: {
-          token: authResponse ? authResponse.access_token : undefined,
-          id: user.getId() || undefined,
+          token: user?.access_token || undefined,
```

Comment:
If `user` is undefined, `user?.access_token` will return `undefined`. Therefore you don't need the or.

Reply:
Fixed